### PR TITLE
WIN compilation error

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
**📝 Description**
Minor compilation error in windows

### CODE REVIEW

File: kratos/solving_strategies/strategies/butcher_tableau.h [hunk 1]
```diff
@@ -66,21 +66,26 @@ namespace Kratos
...
     static constexpr unsigned int Order() {return TOrder;}
     static constexpr unsigned int SubstepCount() {return TSubstepCount; }
 
-    typedef array_1d<double, SubstepCount()> VectorType;
+    // typedef array_1d<double, SubstepCount()> VectorType;
 
     /* Using the following constructs allows us to multiply parts of vectors with parts of matrices
      * while avoiding BOOST's size checks. This is useful to skip multiplications by zero, since
      * for all explicit runge-kutta methods a_ij = 0 for i>j
      */
```
Comment (EduardGomezEscandell) [inline kratos/solving_strategies/strategies/butcher_tableau.h:79]: Can we move this comment up top? It was here to justify the odd choice of Matrix type and now it's a bit disconnected.

File: kratos/solving_strategies/strategies/butcher_tableau.h [hunk 2]
```diff
@@ -66,21 +66,26 @@ namespace Kratos
...
+    typedef std::array<RowType, TSubstepCount-1> MatrixType;
+
     ///@name Type Definitions
     ///@{
 
     static constexpr unsigned int Order() {return TOrder;}
     static constexpr unsigned int SubstepCount() {return TSubstepCount; }
 
-    typedef array_1d<double, SubstepCount()> VectorType;
+    // typedef array_1d<double, SubstepCount()> VectorType;
```
Comment (EduardGomezEscandell) [inline kratos/solving_strategies/strategies/butcher_tableau.h]:
```suggestion
```

File: kratos/solving_strategies/strategies/butcher_tableau.h [hunk 3]
```diff
@@ -66,21 +66,26 @@ namespace Kratos
...
 {
+
 public:
+
+    typedef array_1d<double, TSubstepCount> VectorType;
+    typedef array_1d<double,TSubstepCount-1> RowType;
+    typedef std::array<RowType, TSubstepCount-1> MatrixType;
+
     ///@name Type Definitions
     ///@{
```
Comment (EduardGomezEscandell) [inline kratos/solving_strategies/strategies/butcher_tableau.h]:
```suggestion
    ///@name Type Definitions
    ///@{

    typedef array_1d<double, TSubstepCount> VectorType;
    typedef array_1d<double,TSubstepCount-1> RowType;
    typedef std::array<RowType, TSubstepCount-1> MatrixType;
```

This way they are in the Type definitions section

**Review by EduardGomezEscandell — CHANGES_REQUESTED** [review]
Agree with the fix, minor changes.

**Comment by EduardGomezEscandell** [comment]
It's weird this didn't get caught by CI

File: kratos/solving_strategies/strategies/butcher_tableau.h (hunk 1 above)
Comment (AlejandroCornejo) [reply kratos/solving_strategies/strategies/butcher_tableau.h:79]: please, modify as required, I was only propposing some changes :)

**Review by EduardGomezEscandell — APPROVED** [review]

**Pull request merged**